mhs/common/mhs_common/workflow/__init__.py

```diff
@@ -29,5 +33,6 @@ def get_workflow_map(party_key: str = None, persistence_store: persistence_adapt
         ASYNC_EXPRESS: AsynchronousExpressWorkflow(party_key, persistence_store, transmission, queue_adaptor),
         ASYNC_RELIABLE: AsynchronousReliableWorkflow(),
         FORWARD_RELIABLE: IntermediaryReliableWorkflow(),
+        SYNC_ASYNC: SyncAsyncWorkflow(party_key, transmission = transmission, sync_async_store=persistence_store),
```

Comment:
This may potentially need to change when working on the outbound workflow, but that depends on what approach is taken in that task.